src/commands/remove.js

```diff
@@ -209,24 +229,22 @@ export default async function main(ctx) {
 
     if (confirmation !== 'y' && confirmation !== 'yes') {
       output.log('Aborted');
-      now.close();
-
-      // Hangs otherwise
-      process.exit(1);
```

Comment:
Removed comment and explicit `process.exit()` because they are no longer necessary